java/ql/integration-tests/all-platforms/java/properties-files/weak-hashing/src/main/java/com/example/WeakHashing.java

```diff
@@ -0,0 +1,16 @@
+package com.example;
+
+import java.utils.Properties;
```

Comment:
Your integration test is failing because of this import:

```
[2023-08-23 18:21:09] [build-stdout] [2023-08-23 18:21:09] [autobuild] > Task :compileJava FAILED
[2023-08-23 18:21:09] [build-stdout] [2023-08-23 18:21:09] [autobuild] (...) ql/java/ql/integration-tests/all-platforms/java/properties-files/weak-hashing/src/main/java/com/example/WeakHashing.java:3: error: package java.utils does not exist
[2023-08-23 18:21:09] [build-stdout] [2023-08-23 18:21:09] [autobuild] import java.utils.Properties;
[2023-08-23 18:21:09] [build-stdout] [2023-08-23 18:21:09] [autobuild]                  ^
[2023-08-23 18:21:09] [build-stdout] [2023-08-23 18:21:09] [autobuild] 1 error
[2023-08-23 18:21:09] [build-stdout] [2023-08-23 18:21:09] [autobuild] FAILURE: Build failed with an exception.
```

It should be `java.util.Properties`.

```suggestion
import java.util.Properties;
```